management-api-agent-dse-6.8/pom.xml

```diff
@@ -45,9 +45,27 @@
     </dependency>
     <dependency>
       <groupId>com.datastax.dse</groupId>
-      <artifactId>dse-commons</artifactId>
+      <artifactId>dse-core</artifactId>
```

Comment:
@emerkle826 You might have more insight to review this dependency change. I don't see in this PR why this was done.

Reply:
@burmanm This dependency change was made to allow for access to class `com.datastax.bdp.transport.common.DseReloadableTrustManager`. `dse-commons` provides 4 classes in package `com.datastax.bdp.transport.common`, but not `DseReloadableTrustManager`. I need to double check to make sure that the classes in `dse-commons` are also in `dse-core`.

Reply:
Also, since these dependencies are listed as provided, we should be okay here as we would have bigger issues if `dse-commons` or `dse-core` are not available within the JVM running DSE.

Reply:
issue:
@Miles-Garnsey I double checked and there are classes in `dse-commons` that are _not_ in `dse-core`, though both artifacts provide classes in some of the same packages. That is something that should probably be addressed in the DSE code base, but that outside the scope here.

We should probably keep the `dse-commons` dependency and just add `dse-core` as you have it here.

Reply:
@emerkle826 is there some reason why we need both core and commons? It doesn't seem that anything from core is used which isn't in commons, and I don't know how they'll play together given both artifacts contain the same class names....

Reply:
@Miles-Garnsey we don't need both. I've tested it a little, and since all of these dependencies are marked as `provided`, we would only need them if we are referencing classes defined within them. We do need `dse-db`, and your chnages here require `dse-core`, so you don't have to bring back `dse-common`